src/index.html

```diff
@@ -6,9 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com ">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="image__link" href="#">
+        <img
+        class="image"
+        src="./images/logo.png"
+        alt="logo_MOYO"
+        >
```

Comment:
```suggestion
        <img
          class="image"
          src="./images/logo.png"
          alt="logo_MOYO"
        >
```
Consider fixing indents according to the code style guide